undertow/src/main/java/org/wildfly/extension/undertow/HostAdd.java

```diff
@@ -60,7 +63,9 @@
     static final HostAdd INSTANCE = new HostAdd();
 
     private HostAdd() {
-        super(HostDefinition.ALIAS, HostDefinition.DEFAULT_WEB_MODULE, HostDefinition.DEFAULT_RESPONSE_CODE, HostDefinition.DISABLE_CONSOLE_REDIRECT, HostDefinition.QUEUE_REQUESTS_ON_START);
+        super(HostDefinition.ALIAS, HostDefinition.DEFAULT_WEB_MODULE, HostDefinition.DEFAULT_RESPONSE_CODE,
```

Comment:
`super(HostDefinition.ATTRIBUTES);`

Reply:
I just added the new attribute to the existing list, but I can modify HostDefinition.ATTRIBUTES and use that.